pyriemann_qiskit/utils/distance.py

```diff
@@ -2,28 +2,46 @@
 from docplex.mp.model import Model
 from pyriemann_qiskit.utils.docplex import ClassicalOptimizer, get_global_optimizer
 from pyriemann.classification import MDM
-from pyriemann.utils.distance import distance_functions
+from pyriemann.utils.distance import distance_functions, distance_logeuclid
 from pyriemann.utils.base import logm
+from pyriemann.utils.mean import mean_logeuclid
+from typing_extensions import deprecated
 
 
-def logeucl_dist_convex(X, y, optimizer=ClassicalOptimizer()):
-    """Convex formulation of the MDM algorithm with log-Euclidean metric.
+@deprecated(
+    "logeucl_dist_convex is deprecated and will be removed in 0.3.0; "
+    "please use distance_logeuclid_cpm."
+)
+def logeucl_dist_convex():
+    pass
+
+
+def distance_logeuclid_cpm(A, B, optimizer=ClassicalOptimizer()):
```

Comment:
We should keep the possibility to output weights for backward-compatibility,
but, following the name and the description of the function, it should by default output a distance.
```suggestion
def distance_logeuclid_cpm(A, B, optimizer=ClassicalOptimizer(), return_weights=False):
```